### PR TITLE
fix: second line of userBadge being cut off

### LINKED ISSUE #33733: Bottom of URL path is being cut off in Replay Index
See: 
image

This seems more of an issue with the component we are using.

### PR DESCRIPTION
**Pull request by Danondso**
# Description
Closes #33733 
Screenshots show the line height fix and what the userbadge looks like on the side menu.
# Screenshots
This is the fix with `line-height: normal;`: 
image

This is after with only the line height within the replays index modified.
image

----

By submitting this pull request, I confirm that Sentry can use, modify, copy, and redistribute this contribution, under Sentry's choice of terms.

### CODE REVIEW

**Comment by billyvg** [comment]
@Danondso is this up-to-date w/ master?

**Comment by Danondso** [comment]
> @Danondso is this up-to-date w/ master?

Sorry it musn't have been when I posted it, I just updated it.

**Review by Jesse-Box — COMMENTED** [review]
Can I see a before and after of the change? (screenshots)

**Comment by billyvg** [comment]
@Danondso Can we implement the fix without changing the line-height? It's also affecting the layout of the component in other parts of the app (where it does not get cut off). (See https://storage.googleapis.com/sentry-visual-snapshots/getsentry/sentry/c02777af1c94fc20c7706557f85f33dcd41ee89d/index.html)

File: static/app/views/replays/replays.tsx [hunk 1]
```diff
@@ -251,6 +251,9 @@ const ReplayUserBadge = styled(UserBadge)`
   font-size: ${p => p.theme.fontSizeLarge};
   font-weight: 400;
   line-height: 1.2;
+  div[class*='StyledEmail'] {
```
Comment (billyvg) [inline static/app/views/replays/replays.tsx:254]: This won't work in production as the component names are only there in development. You'll need to export `StyledEmail` (make sure leave a comment why you're exporting it) and import here where you can use it as a selector:

```suggestion
  ${StyledEmail} {
```

Comment (Danondso) [reply static/app/views/replays/replays.tsx:254]: Ah didn't know that, I'll get that adjusted.

**Comment by Danondso** [comment]
@Jesse-Box can I get some design feedback on the change here? This cutting off manifests itself within the userBadge component itself and isn't present on just the replays index (I added before and after screenshots at the top). Let me know how I need to proceed here.

File: static/app/views/replays/replays.tsx (hunk 1 above)
Comment (Danondso) [reply static/app/views/replays/replays.tsx:254]: Actually if I'm waiting for Jesse to give feedback, is it worth doing this yet? I feel like it's prudent to hold off until I get feedback.

**Comment by ryan953** [comment]
Want to make the point that overriding styles with `styled(UserBadge)` seems like an uphill battle. If there are other style tweaks we should consider making them a prop on <UserBadge> itself for example.

**Review by Jesse-Box — APPROVED** [review]
The change looks good!

**Comment by Danondso** [comment]
> The change looks good!

Just to clarify here. Setting line-height: normal within the userBadge does fix the line height for all instances of the userBadge component. I tried to make that clear in the screenshots but wanted to call it out again just in case. Let me know if we're still good with only the replays index being fixed this time around.

File: static/app/views/replays/replays.tsx (hunk 1 above)
Comment (Danondso) [reply static/app/views/replays/replays.tsx]: I went ahead and updated this for now, if I have to roll it back no big deal I'd rather have the all the feedback addressed now.

**Comment by billyvg** [comment]
@Danondso let's apply `line-height` to the `UserBadge` so that it's fixed everywhere.

**Review by billyvg — APPROVED** [review]

**Review by ryan953 — APPROVED** [review]
Fix all the instances. nice

The issue and fix are even visible in the storybook examples:

broken: 
Screen Shot 2022-04-28 at 10 48 33 AM

fixed:
Screen Shot 2022-04-28 at 10 48 58 AM

**Pull request merged**